modules/install-openshift-common-terms.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/index.adoc
+
+:_content-type: REFERENCE
+[id="install-openshift-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the installation content. These terms help you understand installation effectively.
+
+Bootstrap node::
+A temporary machine that runs a minimal Kubernetes configuration to deploy the {product-title} control plane. It is deleted at the end of the installation.
```

Comment:
> It is deleted at the end of the installation.
for IPI, it will be deleted after bootstrap process successfully complete.
for UPI, it could be safely deleted after bootstrap process successfully complete by user.

Reply:
Removed the second sentence to give clarity